ec/src/models/bls12/mod.rs

```diff
@@ -51,6 +58,33 @@ pub struct Bls12<P: Bls12Parameters>(PhantomData<fn() -> P>);
 
 impl<P: Bls12Parameters> Bls12<P> {
     // Evaluate the line function at point p.
+    #[cfg(feature = "parallel")]
+    fn ell(
```

Comment:
Would the parallel case work if it used the same the function as the non-parallel case, but the parallel miller loop caller did `f.clone()`?

Reply:
unfortunatelly no. I tried hard to keep the existing `ell` but went in rabbitholes that ended nowhere.

Reply:
solved